Gems/ImGui/Code/Source/LYCommonMenu/ImGuiInputMonitor.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#ifdef IMGUI_ENABLED
+
+#include <AzCore/std/sort.h>
+#include <AzFramework/Input/Buses/Requests/InputDeviceRequestBus.h>
+#include <AzFramework/Input/Devices/InputDevice.h>
+
+#include "ImGuiInputMonitor.h"
+
+namespace ImGui
+{
+    ImGuiInputMonitor::ImGuiInputMonitor()
+        : m_enabled(false)
+    {
+    }
+
+    void ImGuiInputMonitor::Initialize()
+    {
+        // Connect to EBUSes
+        AZ::TickBus::Handler::BusConnect();
+    }
+
+    void ImGuiInputMonitor::Shutdown()
+    {
+        AZ::TickBus::Handler::BusDisconnect();
+    }
```

Comment:
can we remove the TickBus code from this? OnTick seems to be empty
```suggestion
    void ImGuiInputMonitor::Initialize()
    {
    }

    void ImGuiInputMonitor::Shutdown()
    {
    }
```